client/src/component/User.css

```diff
@@ -0,0 +1,27 @@
+.user-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #d9d9d9;
+  }
+  
+  .user-input {
+    margin-bottom: 10px;
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 14px;
+  }
+  
+  
+  
+  .user-button {
```

Comment:
Keep an eye on formatting, no need for 3 line breaks here

Reply:
ok